modules/layers/src/column-layer/column-layer.js

```diff
@@ -20,14 +20,17 @@
 
 import {Layer, log, createIterable} from '@deck.gl/core';
 import GL from '@luma.gl/constants';
-import {Model, CylinderGeometry, fp64, PhongMaterial} from '@luma.gl/core';
+import {Model, fp64, PhongMaterial} from '@luma.gl/core';
+import ColumnGeometry from './column-geometry';
 const {fp64LowPart} = fp64;
 const defaultMaterial = new PhongMaterial();
 
 import vs from './column-layer-vertex.glsl';
 import fs from './column-layer-fragment.glsl';
 
 const DEFAULT_COLOR = [255, 0, 255, 255];
+const FILL_MODE = 'filled';
+const OUTLINE_MODE = 'stroke';
```

Comment:
Move these two constant modes to `ColumnGeometry` and import from there.

Also call it `STROKE_MODE`